### ISSUE TITLE
PyQt6 support

### ISSUE THREAD

**Issue by arsanysamuel** [agronholm/apscheduler#739]
### Things to check first

- [X] I have searched the existing issues and didn't find my feature already requested there

### Feature description

I'm using QtScheduler with a PyQt6 application and it requires the installation of PyQt5 or Pyside2 in order to work.
```
ImportError: QtScheduler requires either PyQt5, PyQt4, PySide6, PySide2 or PySide installed
```

### Use case

It should support PyQt6 instead of PyQt4 since its support is being dropped, and installing PyQt5 for just running APScheduler will create extra unnecessary dependencies.

**Comment by agronholm** [comment]
I suppose this is reasonable. But is there a reason why you're not using PySide6? Does PyQT6 have something better in comparison?

**Comment by arsanysamuel** [comment]
@agronholm I've just done some searching and it appears that the only difference is the license, I might switch to PySide6 since it's licensed under LGPL and it's almost identical to PyQt6, but having PyQt6 support in APScheduler might be beneficial for some use cases.
You may close the issue if it's unnecessary or not in the plan.

**Comment by agronholm** [comment]
Fixed via f779f48e92eb568d7f38f7d2092acd41be6cf7ee.
